change/@fluentui-react-native-experimental-icon-1d11f2dd-ee78-4145-9bbf-b89f38540903.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "...",
```

Comment:
Improve comment please. Also, there are two empty CHANGELOG files below that shouldn't be here

Reply:
I'' remote it. It, also, was generated